style.css

```diff
@@ -1,3 +1,80 @@
-#root {
-  color: red;
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #e3ded9;
 }
+
```

Comment:
You may like to add more styling in general and try to ensure a better mobile view